synapse/lib/ast.py

```diff
@@ -1572,8 +1610,9 @@ class PropPivot(PivotOper):
             if self.isjoin:
                 yield node, path
 
-            valu = await self.kids[0].compute(path)
-            if valu is None:
+            try:
+                valu = await self.kids[0].compute(path)
+            except s_exc.NoPropValu:
```

Comment:
GOBACK: this is to emulate the existing behavior.  however, it's debatable weather pivoting from an explicit prop should error or not when the prop is not present.